client/src/curriculum/landing.tsx

```diff
@@ -0,0 +1,135 @@
+import { ReactComponent as LandingSVG } from "../../public/assets/curriculum/cur-landing-top.svg";
+import { ReactComponent as LandingStairwaySVG1 } from "../../public/assets/curriculum/cur-landing-stairway-1.svg";
+import { ReactComponent as LandingStairwaySVG2 } from "../../public/assets/curriculum/cur-landing-stairway-2.svg";
+import { ReactComponent as LandingStairwaySVG2Small } from "../../public/assets/curriculum/cur-landing-stairway-2-small.svg";
+import { HydrationData } from "../../../libs/types/hydration";
+import { CurriculumDoc, CurriculumData } from "../../../libs/types/curriculum";
+import { ModulesListList } from "./modules-list";
+import { useCurriculumDoc } from "./utils";
+import { RenderCurriculumBody } from "./body";
+import { useMemo } from "react";
+import { DisplayH2 } from "../document/ingredients/utils";
+import { CurriculumLayout } from "./layout";
+
+import "./index.scss";
+import "./landing.scss";
+
+export function CurriculumLanding(appProps: HydrationData<any, CurriculumDoc>) {
+  const doc = useCurriculumDoc(appProps as CurriculumData);
+  return (
+    <CurriculumLayout
+      doc={doc}
+      withSidebar={false}
+      extraClasses={["curriculum-landing"]}
+    >
+      <RenderCurriculumBody
+        doc={doc}
+        renderer={(section, i) => {
+          if (i === 0) {
+            return (
+              <Header
+                section={section}
+                key={section.value.id}
+                h1={doc?.title}
+              />
+            );
+          }
+          if (section.value.id === "about_the_curriculum") {
+            return About({ section });
+          }
+          if (section.value.id === "modules") {
+            const { title, titleAsText, id } = section.value as any;
```

Comment:
Not great, effenctively turns off typescript for the rest of this block. The titleAsText issue will need fixing though:

```suggestion
            const { title, titleAsText, id } = section.value;
```